doc/source/ray-core/actors/actor-utils.rst

```diff
@@ -0,0 +1,63 @@
+Utility Classes
+===============
+
+Actor Pool
+~~~~~~~~~~
+
+.. tabbed:: Python
+
+    The ``ray.util`` module contains a utility class, ``ActorPool``.
+    This class is similar to multiprocessing.Pool and lets you schedule Ray tasks over a fixed pool of actors.
+
+    .. code-block:: python
+
+        from ray.util import ActorPool
+
+        @ray.remote
```

Comment:
@ericl can we move the snippets that _can_ be tested to a Python file in `doc_code` and pick this up by CI?

I think it's very important to make a habit of this, otherwise we will run into outdated code at some point.

Not sure how to deal with Java and C++ here, what do you think?